### PR TITLE
create Book entity

### PR DESCRIPTION
**Pull request by BohdanTrue**
- created book model
- created book service
- created book repository
- added dependencies

### CODE REVIEW

File: src/main/java/mate/academy/bookstore/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String author;
+    private String isbn;
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
```
Comment (bhdnchui) [inline src/main/java/mate/academy/bookstore/model/Book.java]: Please, pay attention that your entity fields should have more constraints:
image

File: src/main/java/mate/academy/bookstore/repository/BookRepository.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+import java.util.List;
+import mate.academy.bookstore.model.Book;
+
+public interface BookRepository {
+    Book save(Book book);
+
+    List<Book> findAll();
+}
+
```
Comment (bhdnchui) [inline src/main/java/mate/academy/bookstore/repository/BookRepository.java]:
```suggestion
```

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1 +1,7 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/book_store
+spring.datasource.username=root
+spring.datasource.password=apaKanamalaka4a@
```
Comment (bhdnchui) [inline src/main/resources/application.properties:3]: I hope it is not your real password for a lot of your accounts

**Review by bhdnchui — CHANGES_REQUESTED** [review]
Good job! Just a few points to consider.
Let's create local checkstyle.xml file in your project, it is important. 
Also let's think about more suitable PR title.

File: src/main/resources/application.properties (hunk 3 above)
Comment (BohdanTrue) [reply src/main/resources/application.properties:3]: nope

**Review by d1sam — APPROVED** [review]
well done!

**Pull request merged**